lib/elixir/lib/kernel.ex

```diff
@@ -199,7 +199,7 @@ defmodule Kernel do
 
   This means **comparisons in Elixir are structural**, as it has the goal
   of comparing data types as efficiently as possible to create flexible
-  and perform data structures. This distinction is specially important
+  and performant data structures. This distinction is specially important
```

Comment:
Not 100% sure, but maybe "efficient" would be better here?

Reply:
We use "efficiently" a few words before this, so I think "performant" sounds better here. 👍